Casks/tm-elixir.rb

```diff
@@ -0,0 +1,11 @@
+cask "tm-elixir" do
+  version :latest
+  sha256 :no_check
+
+  url "https://github.com/elixir-lang/elixir-tmbundle/archive/master.tar.gz"
+
+  name "A TextMate Bundle for the Elixir programming language."
+  homepage "https://github.com/elixir-lang/elixir-tmbundle"
+
+  tmbundle "Elixir.tmbundle"
```

Comment:
This line causes the following error:

> ==> Installing Cask tm-elixir
> Error: It seems the TextMate bundle source '/usr/local/Caskroom/tm-elixir/latest/Elixir.tmbundle' is not there.
> Error: Install incomplete.

The reason for this is that extracting `master.tar.gz` results in a directory named `elixir-tmbundle-master` while Homebrew looks for something called `Elixir.tmbundle`, which is obviously not there.

To fix this, you can just edit this line so it says:

    tmbundle "elixir-tmbundle-master", target: "Elixir.tmbundle"

That way, Homebrew will find the source directory by the odd name. Also, the `:target` option will instruct it to rename the directory to `Elixir.tmbundle` proper inside TextMate’s `Bundles` directory.